community/cypher/cypher-logical-plans-3.4/src/main/scala/org/neo4j/cypher/internal/v3_4/logical/plans/SingleRow.scala

```diff
@@ -21,10 +21,13 @@ package org.neo4j.cypher.internal.v3_4.logical.plans
 
 import org.neo4j.cypher.internal.ir.v3_4.{CardinalityEstimation, IdName, PlannerQuery}
 
+/**
+  * Produce a single row with the contents of argument
```

Comment:
This one actually should not include args. If it does, there is no point in having both SingleRow and Argument, since they do the same thing. I know the implementation is doing this but it's wrong. :)

Reply:
I agree that they are currently conflicting in purpose. Let's fix that in a separate chunk of work.